packages/sheriff-constants/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@sherifforg/constants",
   "description": "Sheriff constants.",
-  "private": true,
+  "version": "0.0.0",
```

Comment:
Was this intentional?

Also, at this point, why don't you just drop these packages locally too? That'd simplify a lot.

Reply:
> Was this intentional?
Yeah, all of this was intentional. Changesets has strange bahviors. `private: true` broke everything so i had to remove it

> Also, at this point, why don't you just drop these packages locally too? That'd simplify a lot.
Still need these packages because they are shared between the config, the CLI and the website.

Reply:
> `private: true` broke everything so i had to remove it

Huh. That's weird.

> Still need these packages because they are shared between the config, the CLI and the website.

I thought the website depended on the config, but I guess not.